txpool/tx_pool_test.go

```diff
@@ -40,6 +43,177 @@ func newPool(limit int, limitPerAccount int) *TxPool {
 		MaxLifetime:     time.Hour,
 	})
 }
+
+func newPoolWithParams(limit int, limitPerAccount int, BlocklistCacheFilePath string, BlocklistFetchURL string, timestamp uint64) *TxPool {
+	db := muxdb.NewMem()
+	gene := genesis.NewDevnet(timestamp)
```

Comment:
```suggestion
	gene := new(genesis.Builder).
		GasLimit(thor.InitialGasLimit).
		Timestamp(timestamp).
		State(func(state *state.State) error {
			bal, _ := new(big.Int).SetString("1000000000000000000000000000", 10)
			for _, acc := range genesis.DevAccounts() {
				state.SetBalance(acc.Address, bal)
				state.SetEnergy(acc.Address, bal, timestamp)
			}
			return nil
		})
```
This one if fine for this test, so I would suggest no need to change on `genesis.NewDevnet`